imports/api/accounts/server/methods.ts

```diff
@@ -30,45 +27,60 @@ Meteor.methods({
     }
   },
 
-  // "accounts.getAccountSummary": async function (address) {
-  //   let accountList = Accounts.find().fetch();
-  //   console.log(accountList);
-  //   let acc = "0x4caEEFF39cd3b889462b995bDAf2dF97836f490C";
-
-  //   const accounts = await web3.eth.getAccounts();
-  //   const accountsWrapper = await kit.contracts.getAccounts();
-
-  //   // console.log(accounts);
-  //   // console.log(accountsWrapper);
-
-  //   let totalBalance = await kit.getTotalBalance(acc);
-  //   console.log(totalBalance);
-
-
-  //   // const account = await kit.contracts.getAccounts();
-  //   // for (let c in accountList){
-  //   //   if(account)
-  //   // }
-  //   // account.getAccountSummary(acc).then((accountSummary) => {
-  //   //   console.log("!!!!account summary");
-  //   //   console.log(accountSummary);
-  //   // });
-
-  //   const accounts2 = await web3.eth.getAccounts();
-  //   let account = accounts2[0];
-
-  //   console.log(account);
-  //   // if (parseInt(balance) > 0) { 
-  //   //   // update or insert address if balance larger than 0
-  //   //   Accounts.upsert(
-  //   //     { address: address },
-  //   //     { $set: { address: address, balance: parseInt(balance) } },
-  //   //     (error, result) => {
-  //   //       PUB.pubsub.publish(PUB.ACCOUNT_ADDED, {
-  //   //         accountAdded: { address: address, balance: balance },
-  //   //       });
-  //   //     }
-  //   //   );
-  //   // }
-  // },
+  "accounts.getTotalBalance": async function () {
```

Comment:
I suggest this to be done in pages. Imagine if there are thousands or millions of accounts. It can't load and return everything like that.

Reply:
You're right, the previous approach wasn't good. I have moved it to `accounts.update` function so now it will be called after the `Transfer` contract